### PR TITLE
Add FAQ to settings-sync

### PR DESCRIPTION
**Pull request by axonasif**
For https://discord.com/channels/816244985187008514/1029372280117153872

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2885"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2885"></a>

### CODE REVIEW

File: gitpod/docs/references/ides-and-editors/settings-sync.md [hunk 1]
```diff
@@ -84,3 +84,14 @@ Settings in VS Code Desktop might differ from your remote settings, follow the i
 <img class="shadow-medium w-full rounded-xl max-w-3xl mt-x-small" alt="Merge or replace VS Code Settings" src="/images/editors/resolve-merge-conflicts.png">
     <figcaption>Merge or replace VS Code Settings</figcaption>
 </figure>
+
+## FAQs
+
+### How do I disable Gitpod setting sync?
+
+Search for `settings sync off` in your VSCode [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and hit enter.
```
Comment (filiptronicek) [inline gitpod/docs/references/ides-and-editors/settings-sync.md:92]: > and hit enter

We could leverage the [`Keybind` component](https://github.com/gitpod-io/website/blob/main/src/lib/components/keybind.svelte) here to make this more visually pleasing and accessible. WDYT? Should be as simple as importing it on the top of this doc and wrap the keybind in the component 

```suggestion
Search for `settings sync off` in your VSCode [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and hit <Keybind>Enter</Keybind>.
```

**Review by filiptronicek — COMMENTED** [review]
Thanks a lot, @axonasif 🧡! Left a couple suggestions

File: gitpod/docs/references/ides-and-editors/settings-sync.md (hunk 1 above)
Comment (axonasif) [reply gitpod/docs/references/ides-and-editors/settings-sync.md]: Can't commit from Github, will try manually.

Comment (axonasif) [reply gitpod/docs/references/ides-and-editors/settings-sync.md]: Done.

Comment (axonasif) [reply gitpod/docs/references/ides-and-editors/settings-sync.md]: Keybind seems to be missing, reverted.

**Comment by axonasif** [comment]
Hey @filiptronicek, thanks for reviewing 🙏 . Can we merge it now? Since you're the only reviewer so I'm leaving it upto you to press that button 😁  or I can do it too if you say.

**Comment by filiptronicek** [comment]
Pushed another commit @axonasif, if it works for you, let's :shipit:!

**Review by filiptronicek — APPROVED** [review]

**Comment by axonasif** [comment]
Ah, I'm so dumb 😂 
Thanks for fixing that.

**Comment by filiptronicek** [comment]
Don't worry, all good! 🧡

**Pull request merged**